addOns/network/src/main/javahelp/help/contents/options/servercertificates.html

```diff
@@ -6,6 +6,8 @@
 </HEAD>
 <BODY>
 	<H1>Server Certificates</H1>
+	<strong>Note:</strong> This feature is only available in weekly releases and versions after 2.11.
+	<p>
```

Comment:
This `<p>` tag is not closed (same for other additions).

Reply:
Does not need to, can replace with a `br`.